README.md

```diff
@@ -1,49 +1,46 @@
-# [nodejs.org](https://nodejs.org/)
-
-[![CI Status](https://github.com/nodejs/nodejs.org/actions/workflows/ci.yml/badge.svg)](https://github.com/nodejs/nodejs.org/actions/workflows/ci.yml?query=branch%3Amain)
-[![MIT Licensed](https://img.shields.io/badge/license-MIT-blue)](LICENSE)
-[![Crowdin](https://badges.crowdin.net/nodejs-website/localized.svg)](https://crowdin.com/project/nodejs-website)
+<p align="center">
+  <br />
+  <a href="https://nodejs.org">
+    <img src="./public/static/images/logos/nodejs-new-pantone-black.svg" width="200"/>
+  </a>
+</p>
+
+<p align="center">
+  <a href="https://nodejs.org">nodejs.org</a> website built using Nextra (Next.js) with TypeScript, SCSS and MDXv2
+</p>
+
+<p align="center">
+  <a title="CI Status" href="https://github.com/nodejs/nodejs.org/actions/workflows/github-pages.yml?query=branch%3Amain">
+    <img src="https://github.com/nodejs/nodejs.org/actions/workflows/github-pages.yml/badge.svg">
+  </a>
+  <a title="MIT License" href="LICENSE">
+    <img src="https://img.shields.io/badge/license-MIT-blue">
+  </a>
+  <a title="Localised" href="https://crowdin.com/project/nodejs-website">
+    <img src="https://badges.crowdin.net/nodejs-website/localized.svg">
+  </a>
+  <a title="Follow on Twitter" href="https://twitter.com/Nodejs">
+    <img src="https://img.shields.io/twitter/follow/Nodejs.svg?style=social&label=Follow%20@Nodejs">
+  </a>
+  <br />
+  <br />
+</p>
 
 ## What is this repo?
 
-[nodejs.org](https://nodejs.org/) by the [OpenJS Foundation](https://openjsf.org/) builds on the merged community's past website projects to form a self-publishing, community-managed version of the previous site.
-
-On a technical level, inspiration has been taken from the `iojs.org` repo while design and content has been migrated from the old [nodejs.org repo](https://github.com/nodejs/nodejs.org-archive). These technical changes have helped to facilitate community involvement and empower the foundation's internationalization communities to provide alternative website content in other languages.
+[Nodejs.org](https://nodejs.org/) by the [OpenJS Foundation](https://openjsf.org/) builds on the merged community's past website projects to form a self-publishing, community-managed version of the previous site.
```

Comment:
I'm not sure this sentence is needed. The community split referenced by "merged community" was back in 2014/2015.

Reply:
mmmm 🤔